account_bank_statement_import_txt_xlsx/models/account_bank_statement_import_sheet_parser.py

```diff
@@ -56,6 +56,12 @@ def parse(self, mapping, data_file, filename):
             journal.code,
             path.basename(filename),
         )
+        if mapping.skipped_rows > 0 and name.lower().endswith('.csv'):
```

Comment:
If you skip rows in `_parse_lines` why this block is needed?

Reply:
Hi @alexey-pelykh, in _parse_lines() the xls lines are skipped, here is the code for the .csv

Reply:
This is not the best way to detect a csv, which in the end is just a plain text file. Why don't you just skip the proper offset in this `else`: https://github.com/OCA/bank-statement-import/blob/2e6a37f91e21d0d81b0062f7726bd6f427271f31/account_bank_statement_import_txt_xlsx/models/account_bank_statement_import_sheet_parser.py#L123-L128

Something like:
```python
     for i in range(mapping.skipped_rows):
         next(csv_or_xlsx)
     header = [value.strip() for value in next(csv_or_xlsx)]
```